lib/components/sections/featured_section.dart

```diff
@@ -65,7 +65,7 @@ class FeaturedSection extends ConsumerWidget {
     return LayoutBuilder(builder: (context, constraints) {
       final filteredItems = data.items.items.where((e) {
         final episode = e.item.asOrNull<Fragment$ItemSectionItem$item$$Episode>();
-        return episode == null || !episode.locked;
+        return episode?.locked == false;
```

Comment:
This isn't the same as before. If "episode" is null it isn't == false

Reply:
Sorry. I assumed that we show only episodes on the featured section in which case it doesn't make sense to keeps an item if it is not an episode.